lib/slack-profile-cache.js

```diff
@@ -251,3 +252,18 @@ function timeout (ms) {
 async function sleep (timeoutMs) {
   await timeout(timeoutMs)
 }
+
+async function getUsersList (users = [], cursor = '') {
+  try {
+    const res = await slackWebClient.users.list({limit: 200, cursor: cursor})
+    users = users.concat(res.members)
+
+    if (!res.response_metadata.next_cursor) {
+      return users
+    }
+
+    return getUsersList(users, res.response_metadata.next_cursor)
+  } catch (e) {
+    console.log(e)
```

Comment:
We should rethrow the exception here (the top-level catch clause will already use  `robot.log.error` to log the message).

Reply:
Oh I see, I didn't read the code thoroughly.